web-common/src/features/dashboards/stores/dashboard-stores-test-data.ts

```diff
@@ -322,6 +327,32 @@ export function getOffsetByHour(time: Date) {
   );
 }
 
+export function initStateManagers(
```

Comment:
IMO this extra layer makes the code harder to follow – adds indirection.

Reply:
It is a subset of test initialisation used in 2 places. Once we add tests for everything that uses `StateManagers` this would be used there as well.